payment-manager/src/main/java/com/qpay/paymentmanager/config/WebClientConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.qpay.paymentmanager.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+public class WebClientConfig {
+
+
+    @Value("${webclient.base-url}")
```

Comment:
smth like this. I did not check whether it works or not
```
@Configuration
@ConfigurationProperties(prefix = "webclient.services")
@Data
public class WebClientConfig {

    private ServiceUrl notificationManager;

    private ServiceUrl transactionHistory;


    @Data
    public static class ServiceUrl {
        
        private String baseUrl;
        
    }
}
```

Reply:
check it https://www.baeldung.com/configuration-properties-in-spring-boot .
Also instead of class, you can try to use `record` to be more fancy